### ISSUE TITLE
Add instructions to generate project without installing

### ISSUE THREAD

**Issue by MrEmanuel** [MicrosoftDocs/msteams-docs#3651]
TLDR: Generate a teams app project without installing packages globally using the following command: `npm init yo teams`

Generators like this one are not frequently used. Installing the package globally means it will most likely be outdated the next time you want to use it, and updating and keeping track of versions is a bit of a pain. 

A convenient solution is to utilize npm's init-feature. It invokes `npx` which is a package runner. It gets the executable, runs the command, and then gets rid of the executable. This is preferable if you're not using the executable frequently. 

I think it would be better to list these installation instructions instead of the current ones.

Again, the installation instructions should be: 
`npm init yo teams`

Instead of: 
`npm install generator-teams --global`
`yo teams`

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 42e0cad5-f762-c253-7d3b-b23ae3cb5b1c
* Version Independent ID: f1babe27-cbc7-f757-11b4-fc8fa1e36873
* Content: [Tutorial - Create your first app using the Yeoman generator - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/tutorials/get-started-yeoman?WT.mc_id=m365-9410-cxa)
* Content Source: [msteams-platform/tutorials/get-started-yeoman.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/tutorials/get-started-yeoman.md)
* Product: **msteams**
* GitHub Login: @laujan
* Microsoft Alias: **lajanuar**

**Comment by Ravindra-MSFT** [comment]
Hi @MrEmanuel, Thank you for the insights on the document. Taking this for internal discussion with team.

**Comment by Ravindra-MSFT** [comment]
Hi @v-abirade Could you please replace `npm init yo teams` instead of `yo teams` and remove below snip in the documentation.

![image](https://user-images.githubusercontent.com/84447722/122922456-43670900-d381-11eb-827e-f2eb0563a5d6.png)

**Comment by v-vrajaraman** [comment]
Fixed as part of [Tutorials and Code Samples](https://github.com/MicrosoftDocs/msteams-docs/pull/3604/).

**Comment by surbhigupta12** [comment]
Issue closed as part of PR #3651
